styles/mobbile.css

```diff
@@ -1,66 +1,78 @@
-@media only screen and (max-width: 768px)
-{
+@media only screen and (max-width: 768px) {
```

Comment:
I made a number of formatting changes to your CSS. Notice that opening `{` are on the same line as selectors, and that nested declarations are indented to the same level